x-pack/test/security_solution_api_integration/test_suites/detections_response/basic_license/rule_management/coverage_overview.ts

```diff
@@ -0,0 +1,639 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+
+import {
+  RULE_MANAGEMENT_COVERAGE_OVERVIEW_URL,
+  ThreatArray,
+} from '@kbn/security-solution-plugin/common/api/detection_engine';
+import { FtrProviderContext } from '../../../../ftr_provider_context';
+import {
+  createPrebuiltRuleAssetSavedObjects,
+  createRuleAssetSavedObject,
+  createRule,
+  deleteAllRules,
+  getSimpleRule,
+  installPrebuiltRulesAndTimelines,
+  installPrebuiltRules,
+  createNonSecurityRule,
+} from '../../utils';
+
+export default ({ getService }: FtrProviderContext): void => {
+  const supertest = getService('supertest');
+  const log = getService('log');
+  const es = getService('es');
+
+  describe('@serverless @ess coverage_overview', () => {
+    beforeEach(async () => {
+      await deleteAllRules(supertest, log);
+    });
+
+    it('does NOT error when there are no security rules', async () => {
+      await createNonSecurityRule(supertest);
+      const rule1 = await createRule(supertest, log, {
+        ...getSimpleRule(),
+        threat: generateThreatArray(1),
+      });
+
+      const { body } = await supertest
+        .post(RULE_MANAGEMENT_COVERAGE_OVERVIEW_URL)
+        .set('kbn-xsrf', 'true')
+        .set('elastic-api-version', '1')
+        .set('x-elastic-internal-origin', 'foo')
+        .send({})
+        .expect(200);
```

Comment:
Looks like you can extract this call as a utility function which only takes the filter as parameter. Would save a lot of repeated code in this file and make it more concise and easier to understand. Something like:
```ts
export const getCoverageOverview = async (
  supertest: SuperTest.SuperTest<SuperTest.Test>,
  filters: FiltersType (?)
): Promise<InstallPrebuiltRulesAndTimelinesResponse> => {
  const response = await supertest
      .post(RULE_MANAGEMENT_COVERAGE_OVERVIEW_URL)
      .set('kbn-xsrf', 'true')
      .set('elastic-api-version', '1')
      .set('x-elastic-internal-origin', 'foo')
      .send(filters)
      .expect(200)

  return response.body;
};
```